precompile/bank/IBank.sol

```diff
@@ -12,8 +14,7 @@ interface IBank {
     function send(
         address fromAddress,
         address toAddress,
-        string calldata denom,
-        uint256 amount
+        Coin[] memory funds
```

Comment:
We are using the same variable names as cosmos. All other modules use amount, and only wasm uses funds. I think it would be better to match them to avoid confusion later.

https://github.com/cosmos/cosmos-sdk/blob/acb3d747609213d1ebce4236d194580c47534468/x/bank/types/tx.pb.go#L37-L41

Reply:
[4e9faa3](https://github.com/xpladev/xpla/pull/166/commits/4e9faa3d01a8711d505bad602e9b4f0b5a932178)

Reply:
wasm module use funds

https://github.com/CosmWasm/wasmd/blob/c498a467f697b05477b9ae06c5f5d888566d3533/x/wasm/types/tx.pb.go#L350

Reply:
[f509a7b](https://github.com/xpladev/xpla/pull/166/commits/f509a7b4140b6560dcf36a7af4d50cf60193494a)